omero/developers/Model/EveryObject.txt

```diff
@@ -4,14 +4,6 @@ Glossary of all OMERO Model Objects
 Overview
 --------
 
-In navigating the model objects used by the :doc:`OMERO API
```

Comment:
I was surprised to see it gone, but It was too late for me to do further investigation